src/components/SRNEntry.vue

```diff
@@ -164,64 +179,87 @@ export default {
         },
       ];
     },
+    /* checks to see if the input character is a number. Makes use of ASCII values.
+    @param {Event} - e - event triggered when a character is typed.  */
     isValidNumericEntry(e) {
-      //checking to see if each char typed by user is only a number
       if (e.keyCode >= 48 && e.keyCode <= 57) return true;
       else e.preventDefault();
     },
+    // adds a new dict to the array - userIDList
     addNewEmptyField() {
       this.userIDList.push({ userID: "", valid: false });
     },
+    /* removes an element in the array at a given index. 
+    @param {Number} - index - the index of the input box where - button is clicked. */
     removeInputField(index) {
       this.userIDList.splice(index, 1);
     },
+    // sets the invalid input message to null (default)
     resetInvalidInputMessage() {
       this.invalidInputMessage = "";
     },
+    // sets the invalid login message to null (default)
     resetInvalidLoginMessage() {
       this.invalidLoginMessage = "";
     },
+    // sets the temp valid flag to false (default) for authentication of a possible next user
     resetValidFlag() {
-      this.isUserValid = false;
+      this.isCurrentUserValid = false;
     },
+    // sets the valid key of the latest user to the value of the temp flag. The temp flag contains the value returned by the backend
     setValidFlag() {
-      this.getLatestEntry["valid"] = this.isUserValid;
+      this.getLatestEntry["valid"] = this.isCurrentUserValid;
     },
-    //resets input field at an index
+    //resets the userID key of an element in the array at a particular index.
     resetEntry(index){
         this.userIDList[index]["userID"] = "";
     },
+    /* this function is called whenever the + button is clicked. The most recent typed entry is authenticated against the database. 
```

Comment:
Function comments should be after the function name (next line)

Reply:
Do you mean docstrings?
Docstrings in JS are typed just above the function name